resources/uscore_bundle_patient_85.json

```
@@ -18427,6 +18450,477 @@
+              "text" : "Are you worried about losing your housing?",
+              "answer" : [{
+                "valueCoding" : {
+                  "system" : "http://loinc.org",
+                  "code" : "LA30122-8",
+                  "display" : "I choose not to answer this question"
+                }
+              }]
+            },
+            {
+              "linkId" : "/93042-0/56799-0",
+              "text" : "Address",
+              "answer" : [{
+                "valueString" : "Broadway"
+              }]
+            }]
+          },
+          {
+            "linkId" : "/93041-2",
+            "text" : "Money and resources",
+            "item" : [{
+              "linkId" : "/93041-2/82589-3",
+              "text" : "Highest level of educ",
+              "answer" : [{
+                "valueCoding" : {
+                  "system" : "http://loinc.org",
+                  "code" : "LA30191-3",
+                  "display" : "Less than high school degree"
+                }
+              }]
+            },
+            {
+              "linkId" : "/93041-2/67875-5",
+              "text" : "Employment status current",
+              "answer" : [{
+                "valueCoding" : {
+                  "system" : "http://loinc.org",
+                  "code" : "LA17956-6",
+                  "display" : "Unemployed"
+                }
+              }]
+            },
+            {
+              "linkId" : "/93041-2/76437-3",
+              "text" : "Primary insurance",
+              "answer" : [{
+                "valueCoding" : {
+                  "system" : "http://loinc.org",
+                  "code" : "LA17849-3",
+                  "display" : "Medicaid"
+                }
+              }]
+            },
+            {
+              "linkId" : "/93041-2/63586-2",
+              "text" : "What was your best estimate of the total income of all family members from all sources, before taxes, in last year?",
+              "answer" : [{
+                "valueDecimal" : 25000
+              }]
+            },
+            {
+              "linkId" : "/93041-2/93031-3",
+              "text" : "In the past year, have you or any family members you live with been unable to get any of the following when it was really needed?",
+              "answer" : [{
+                "valueCoding" : {
+                  "system" : "http://loinc.org",
+                  "code" : "LA30126-9",
+                  "display" : "Clothing"
+                }
+              },
+              {
+                "valueCoding" : {
+                  "system" : "http://loinc.org",
+                  "code" : "LA30127-7",
+                  "display" : "Child care"
+                }
+              }]
+            },
+            {
+              "linkId" : "/93041-2/93030-5",
+              "text" : "Has lack of transportation kept you from medical appointments, meetings, work, or from getting things needed for daily living",
```

```suggestion
              "text" : "Has lack of transportation kept you from medical appointments, meetings, work, or from getting things needed for daily living?",
```

Everything currently passes when using the Inferno validator wrapper, but I happened to be working on migrating this test kit to the HL7 validator wrapper and there is one error `If text exists, it must match the questionnaire definition for linkId /93041-2/93030-5`. (You can also verify on the CLI validator, for example `java -jar ~/Downloads/validator_cli_6_3_4.jar -ig "hl7.fhir.us.core#6.1.0" ./resources/uscore_bundle_patient_85.json > 85_errors.txt`) 

I suspect that QR validation was added in a newer version of the core validator than we are using in the Inferno wrapper. From my testing, adding this question mark (to match the Questionnaire item at http://hl7.org/fhir/us/core/STU6.1/Questionnaire-prapare-example.html ) will make it pass in the HL7 validator as well as the Inferno validator.

---

Good catch. All resources are copied from examples in either US Core IG or SDC IG. Will fix this.